client/src/app/queries/applications.ts

```diff
@@ -44,6 +53,27 @@ export const useFetchApplications = () => {
   };
 };
 
+export const ApplicationQueryKey = "application";
+
+export const useFetchApplicationByID = (id: number | string) => {
+  const { data, isLoading, error } = useQuery({
+    queryKey: [ApplicationQueryKey, id],
+    queryFn: () => getApplicationById(id),
```

Comment:
I'm not sure if we're consistent about this elsewhere, but we may want to choose one between `...ByID` and `...ById` and stick with it.

... or not worry about it, because I sent myself down this rabbit hole that I regret: https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Reply:
https://github.com/konveyor/tackle2-ui/issues/1307